### DISCUSSION TITLE
list files in workspace as a tree (much like Neotree).

### DISCUSSION THREAD

**Discussion by madelaney** [nvim-neorg/neorg#1454] [General]
I'm new to neorg, and while I'm getting the hang of it. I would like to see the neorg files (in the current workspace) as a tree view on either the right or left side of the window. Is this something that is possible given the current state of neorg?

**Comment by max397574** [comment]
not currently possible
neotree has some filter options though
perhaps you could work with these

**Comment by pysan3** [comment]
Neo-tree has a plugin system that you can use to implement an external source. 

The API is not well documented tho which I'm trying to manage in the next release...

**Reply by pysan3** [reply]
JFYI one example of an external source is https://github.com/miversen33/netman.nvim
